src/Test.FunctionalTests.Sarif/Multitool/ValidateCommandTests.cs

```diff
@@ -176,6 +176,52 @@ public void SARIF2005_ProvideToolProperties_Valid()
         public void SARIF2005_ProvideToolProperties_Invalid()
             => RunInvalidTestForRule(RuleId.ProvideToolProperties);
 
+        private const string DottedQuadFileVersionAcceptedConfiguration =
```

Comment:
>DottedQuadFileVersionAcceptedConfiguration [](start = 29, length = 42)

I don't like maintain structural code in string literals, as you know. Minimally, your string literals contain references to names like 'DottedQuadFileVersion' that you should be referencing from code (via an interpolated string). Ideally, this code would actually invoke the (easily consumed) property bag machinery to create these instances. That way if the property bag code itself did something (like rename the 'Type' attribute), it would all just work. We aren't really at risk of that, of course.

But please do figure out how to create helpers that directly instantiate the property bags you need. If you require a string literal, of course, find the code that serializes them. This all exists. The property bag unit tests likely have useful patterns or code you can grab.